perception/radar_tracks_msgs_converter/launch/radar_tracks_msgs_converter.launch.xml

```diff
@@ -3,13 +3,13 @@
   <arg name="input/odometry" default="/localization/kinematic_state"/>
   <arg name="output/radar_detected_objects" default="output/radar_detected_objects"/>
   <arg name="output/radar_tracked_objects" default="output/radar_tracked_objects"/>
-  <arg name="param_path" default="$(find-pkg-share radar_tracks_msgs_converter)/config/radar_tracks_msgs_converter.param.yaml"/>
+  <arg name="config_file" default="$(find-pkg-share radar_tracks_msgs_converter)/config/radar_tracks_msgs_converter.param.yaml"/>
```

Comment:
[MUST]
This unnecessary interface changes affect many Autoware projects.
Please revert this change.

Reply:
@kaspermeck-arm 
Could you provide the reasons for this modification?

Like @scepter914 says, this would affect users who includes this radar_tracks_msgs_converter.launch.xml to their own project.
Since we don't want to disturb developers, let's keep this as and focus on node configuration, not launch configuration. If we need to unify the launch arguments as well, I think we should discuss this under https://github.com/autowarefoundation/autoware.universe/issues/3386 first.

Reply:
@mitsudome-r 

To align with the original PR
* https://github.com/autowarefoundation/autoware.universe/pull/4911/files#diff-9328a941a658e3bda840cef66db3784d382a5e82e94560b6b31f6aef57091e81

and because in
* https://github.com/autowarefoundation/autoware.universe/commit/c53b81f3ad7314c53a27b3bc7721aaef6ec35c9d

it wasn't addressed.

I'll update the launch file and we can discuss naming convention when we start the ROS Launch dojo.